loop/e2e-test/.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules
+*.log
+*.dmg
+*.tar*
+*.gz*
+*.zip*
```

Comment:
Similar to [previous comment](https://github.com/mozilla-services/services-test/pull/8/files#r34288985).
Not sure if we want to merge these with the root /.gitignore file and just have one global ignore list for optimal ignoring.

Or add @pdehaan to your .gitignore file. Whatever, @pdehaan don't care.